src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Services/Helpers/DateTimeProviderContext.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Collections;
+using System.Threading;
+
+namespace DevAdventCalendarCompetition.Services
+{
+    public class DateTimeProviderContext : IDisposable
```

Comment:
if you are using someone else's solution, it is good to practise to add a link to it (code or blog post) - so you should probably add https://dvoituron.com/2020/01/22/UnitTest-DateTime/

Reply:
Do you mean in comment in code?

Reply:
yes, sth like
`//based on solution https://dvoituron.com/2020/01/22/UnitTest-DateTime/`